.github/workflows/scripts/docs/gen-docs.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import logging
+import re
+from pathlib import Path
+from typing import Literal
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+logging.basicConfig(level=logging.INFO)
+
+logger = logging.getLogger("doc-gen")
+
+BASEPATH = Path(__file__).resolve().parent
+
+
+def parse_group(group: Literal["calculations", "parameters", "visibilities"]) -> list[dict]:
```

Comment:
Isn't it possible to directly import the code and query the data directly, instead of searching the file with regex?

Reply:
Not sure about that to be honest 🤔
Maybe I'll look into that on Monday.
As mentioned, this is mostly a proof of concept, how we collect the data can be changed later on.